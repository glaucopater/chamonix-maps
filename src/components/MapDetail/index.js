import React from 'react';
import { StyledMapDetail } from './styled';
import Image from '../Image';
import MapPath from '../MapPath';
import strings from '../../constants/strings';

const MapDetail = ({ id, name, skiDifficulty, description, geoData }) => (
  <StyledMapDetail>
    <MapPath geoData={geoData} />
    <figure>
      <Image src={'random'} alt={name} />
      <figcaption>
        <a href={`/map/${id}/`}>
          <h3>
            {name} {' | '} {strings.difficulty}
            {skiDifficulty}
          </h3>
        </a>
      </figcaption>{' '}
    </figure>
    <p>{description}</p>
  </StyledMapDetail>
);

export default MapDetail;
