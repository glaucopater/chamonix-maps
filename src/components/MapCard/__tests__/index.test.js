import { shallow } from 'enzyme';
import React from 'react';
import MapDetail from '../index';

describe('<MapDetail/>', () => {
  it('renders correctly', () => {
    const props = {
      name: '',
      difficulty: '',
      shortDescription: '',
    };
    const wrapper = shallow(<MapDetail {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
