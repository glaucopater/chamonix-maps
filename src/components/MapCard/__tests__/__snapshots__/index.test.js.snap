// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MapDetail/> renders correctly 1`] = `ShallowWrapper {}`;
